hudi-cli/src/main/java/org/apache/hudi/cli/commands/DatasetsCommand.java

```diff
@@ -118,20 +118,21 @@ public String createTable(
 
   @CliAvailabilityIndicator({"desc"})
```

Comment:
may I know, why you did not remove this method similar to other files? sorry, I am new to this. trying to understand some basics

Reply:
> I am new to this. trying to understand some basics

Forgot to delete. ^ ^